components/search/SearchResultsList.stories.js

```diff
@@ -0,0 +1,49 @@
+import { storiesOf } from '@storybook/vue';
+
+import SearchResultsList from './SearchResultsList.vue';
+
+storiesOf('Search page', module)
+  .add('List of search results', () => ({
+    components: { SearchResultsList },
+    data() {
+      return { results: [
+        {
+          edmPreview: 'img/landscape.jpg',
+          fields: [
```

Comment:
`fields` should be an Object like: `{ dcTitle: [], dcDescription: [] }`

Let's also account for the different kinds of fields we may have in each:
* An Object keyed by language to be output raw
* A single-value array to be output as-is
* A multi-value array to be output as a list